### PR TITLE
manual qa documentation updated

### PR DESCRIPTION
**Pull request by mdacoca**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:44]: That is not exactly true.

Reviewers can be assigned by project/task owner/assignee.
And by organization maintainers of course.

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:70]: That is not obligatory. You do not need to save the work in advance. 
Issues are created on the server immediately after submission

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:96]: The same comment about saving

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:135]: Do we need this to be dublicated?

**Comment by bsekachev** [comment]
Maybe we should mention, that review mode is available only for 2D content 
Not available for 3D point clouds

**Comment by mdacoca** [comment]
> Maybe we should mention, that review mode is available only for 2D content Not available for 3D point clouds

It is mentioned at the very beginning of the document.

**Review by bsekachev — APPROVED** [review]

**Review by azhavoro — APPROVED** [review]

**Comment by bsekachev** [comment]
We will merge it after release just in case

**Pull request merged**
